exercises/practice/protein-translation/.docs/instructions.md

```diff
@@ -0,0 +1,45 @@
+# Instructions
+
+Translate RNA sequences into proteins.
+
+RNA can be broken into three nucleotide sequences called codons, and then translated to a polypeptide like so:
+
+RNA: `"AUGUUUUCU"` => translates to
+
+Codons: `"AUG", "UUU", "UCU"`
+=> which become a polypeptide with the following sequence =>
+
+Protein: `"Methionine", "Phenylalanine", "Serine"`
+
+There are 64 codons which in turn correspond to 20 amino acids; however, all of the codon sequences and resulting amino acids are not important in this exercise.
+If it works for one codon, the program should work for all of them.
+However, feel free to expand the list in the test suite to include them all.
+
+There are also three terminating codons (also known as 'STOP' codons); if any of these codons are encountered (by the ribosome), all translation ends and the protein is terminated.
```

Comment:
```suggestion
There are also three terminating codons (also known as 'STOP' codons);
if any of these codons are encountered (by the ribosome), all translation ends and the protein is terminated.
```

One line per sentence, though this can be one line per phrase, since this is a reasonable break, and only one portion of these might be changed or updated in the future, leaving a clean diff at that point.

Definitely an optional change.

Reply:
@kotp You do know that these `instructions.md` file's contents are synced from problem-specifications, right? So any suggestions to change them would have to be done there.